### PR TITLE
Create group rebased

### PR DESCRIPTION
**Pull request by Vitexus**

### CODE REVIEW

File: src/Client/GroupsClient.php [hunk 1]
```diff
@@ -30,6 +30,8 @@ class GroupsClient extends Client implements ClientInterface
     public const ACTION_ENABLE_USER = 'enable_user';
     public const ACTION_RENAME_GROUP = 'rename';
     public const ACTION_CREATE_GROUP = 'create';
+    public const ACTION_LIST_GROUPS = 'list';
```
Comment (OskarStark) [inline src/Client/GroupsClient.php:33]: Private?

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@ composer.lock
 .php-cs-fixer.dist.php
 .idea/
 .build
+.php-version
```
Comment (OskarStark) [inline .gitignore:7]: Can be removed

File: src/Client/AssignLicenseClient.php [hunk 3]
```diff
@@ -24,6 +24,9 @@ class AssignLicenseClient extends Client implements ClientInterface
 {
     public const API_PATH = 'licenses/assign.json';
 
+    /**
+     * {@inheritDoc}
+     */
```
Comment (OskarStark) [inline src/Client/AssignLicenseClient.php:29]: Can be removed

File: tests/Client/GroupsClientTest.php [hunk 4]
```diff
@@ -46,6 +46,8 @@ public function testBuildApiUrl(): void
 
         $client = new GroupsClient($group, GroupsClient::ACTION_ADD_USER);
         $this->assertEquals('https://api.pushover.net/1/groups/eeee5555EEEE5555ffff6666FFFF66/add_user.json?token=cccc3333CCCC3333dddd4444DDDD44', $client->buildApiUrl());
+      	$client = new GroupsClient($group, GroupsClient::ACTION_LIST_GROUPS);
+        $this->assertSame('https://api.pushover.net/1/groups.json?token=cccc3333CCCC3333dddd4444DDDD44', $client->buildApiUrl());
```
Comment (OskarStark) [inline tests/Client/GroupsClientTest.php:50]: Duplicate lines

File: tests/Client/Response/ListGroupsResponseTest.php [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+namespace Client\Response;
+
+use Serhiy\Pushover\Client\Response\ListGroupsResponse;
+use PHPUnit\Framework\TestCase;
+use Serhiy\Pushover\Recipient;
+
+class ListGroupsResponseTest extends TestCase
+{
+    /**
+     * prepare ListGroupsResponse Test
```
Comment (OskarStark) [inline tests/Client/Response/ListGroupsResponseTest.php:21]: Comment can be removed

File: tests/Client/GroupsClientTest.php (hunk 4 above)
Comment (Vitexus) [reply tests/Client/GroupsClientTest.php:51]: Are you sure ?

**Comment by slunak** [comment]
Sorry, I dont understand purpose of this PR... it is called Create Group Rebased, but includes functionality or previous PR with list groups functionality.

File: tests/Client/GroupsClientTest.php (hunk 4 above)
Comment (OskarStark) [reply tests/Client/GroupsClientTest.php:51]: Oh no :-)

**Comment by OskarStark** [comment]
I think we can close this PR and work in 
* #30 

Just merge the master branch in your branch, @slunak can squash all the commits while merging

**Pull request closed without merging**